client/src/components/Tickets/TicketModal.js

```diff
@@ -34,6 +34,23 @@ const formSchema = Yup.object().shape({
   reply: Yup.string().required('Required')
 });
 
+const Attachments = ({ attachments }) => {
+  return (
+    <div>
+      Attachments:
+      <ul>
```

Comment:
If you look in the console, you'll see this warning:
```Warning: Each child in a list should have a unique "key" prop.```
Can you add a unique key to each element? Look at other `.map` methods on the client side for an example